### DISCUSSION TITLE
Why do both REST and GraphQL APIs require authentication?

### DISCUSSION THREAD

**Discussion by adlai** [community/community#37927] [Apps, API and Webhooks] [answered]
I find it bizarre that endpoints more efficient both for your servers and any curious clients are burdened with the requirements of authentication, when anyone unauthenticated can scrape information off the website.

**Comment by jfullstackdev** [comment, marked as answer]
it depends on the endpoint you are using,

for example, this requires authentication 

https://api.github.com/jdevstatic

![image](https://user-images.githubusercontent.com/47092464/199151193-84bdb785-507f-463c-9c61-51a7596f6bba.png)

that's why it's "Not Found"

but this does not require authentication 

https://api.github.com/users/jdevstatic

![image](https://user-images.githubusercontent.com/47092464/199151101-2f401453-d05c-4612-802b-2a075669ff32.png)

this one is public endpoint

**Reply by adlai** [reply]
Where is that documented?

**Reply by jfullstackdev** [reply]
in the API documentation, when it mentions the `authenticated` user, it means, it needs authentication, the other one is just get all public info of a user, which means it does not need authentication

![image](https://user-images.githubusercontent.com/47092464/199190511-0cf9b2bc-c78c-406d-ac62-c5b5e1776859.png)

![image](https://user-images.githubusercontent.com/47092464/199190546-3183e59a-dada-485c-9386-fbb2264106ef.png)

all of it are contained here, it's just an example

https://docs.github.com/en/rest

**Reply by ghostinhershell** [reply]
Hey @adlai   👋

If @jdevstatic's response here answered your question, we'd appreciate you circling back and marking their response as the answer to this post. Otherwise, feel free to follow-up with any additional questions. Cheers!
